website/docs/components/stepper/indicator/partials/specifications/anatomy.md

```diff
@@ -12,7 +12,7 @@
 
 ### Task indicator
 
-![Task Indicator Anatomy](/assets/components/stepper/task-indicator-anatomy.png =226x*)
+![Task Indicator Anatomy](/assets/components/stepper/indicator/task-indicator-anatomy.png =226x*)
```

Comment:
The images should be appropriately sized in Figma before export that you shouldn't need "=226x*"